Pkgfile

```diff
@@ -39,7 +39,7 @@ vars:
   # renovate: datasource=github-releases depName=flannel-io/cni-plugin
   flannel_cni_version: v1.2.0
   flannel_cni_ref: 18a3027e7d03feeb6ecdfdbc3bf254a8c8b38b04
```

Comment:
ref needs updating

Reply:
ah, there was no sha update, it was just extra space which got removed it looks like